packet/accumulator_test.go

```diff
@@ -38,7 +38,7 @@ func ExamplePacketAccumulator() {
 
 	secondPacket, _ := hex.DecodeString("47006411f0002b59bc22ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
 
-	packets := []Packet{firstPacket, secondPacket}
+	packets := [][]byte{firstPacket, secondPacket}
```

Comment:
There are a few changes like this. Changing to `[]byte` from `Packet`. What's the benefit? We lose the compile time checking, which is why the change to `[188]byte` would be so beneficial, IMO.

Reply:
Here it's not a packet, it's a `[]byte`, having been returned by a stdlib function that returns `[]byte`. This change is in tests.